packages/StepTracker/Step/Step.jsx

```diff
@@ -1,30 +1,35 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import classNames from 'classnames'
+
+import Icon from '@tds/core-decorative-icon'
+
+import styles from '../StepTracker.modules.scss'
 
 /**
  * A single step of a StepTracker.
- *
- * _This component can only be used as a child of `StepTracker`, and must be accessed as a
- * name-spaced component: `StepTracker.Step`._
- *
- * @see See [StepTracker](#steptracker) for usage.
  */
-const Step = ({ label, status, stepNumber }) => {
-  const cls = classNames('step-tracker__step', {
-    'step-tracker__step--completed': status === 'completed',
-    'step-tracker__step--processing': status === 'processing',
-    'step-tracker__step--waiting': status === 'waiting',
-  })
+
+const Step = ({ label, status, stepNumber, stepIndex }) => {
+  const cls = [styles.step]
```

Comment:
A more conventional pattern with CSS modules is to use the `composes` keyword in the CSS file. So the `completed` and `processing` classes should "compose" the `step` class. Then you won't have to do this array manipulation.

```css
.step {
 // styles
}

.completed {
  composes: step;

  // completed styles
}

.processing {
  composes: step;

  // processing styles
}
```